### ISSUE TITLE
Keymap to jump between [Response | Headers | Cookies | Stats]

### ISSUE THREAD

**Issue by h3ndry** [rest-nvim/rest.nvim#315]
What is the default keymap to jump between *response* , *headers* without using mouse

**Comment by NTBBloodbath** [comment]
Hi, the results window has a winbar (`:h 'winbar'`) that in the upper right corner has a hint to use `?` which opens a small floating window with all the existing keybinds for the results window. Note that `?` must be used when focusing the results window as otherwise it won't work, this is made by design as I didn't want it to clash with the default Neovim's `?` keybind to search in the buffer.

tl;dr
- `H` - previous pane
- `L` - next pane
![image](https://github.com/rest-nvim/rest.nvim/assets/36456999/92dd6749-9a97-4b1d-b682-0855c1a6be50)
